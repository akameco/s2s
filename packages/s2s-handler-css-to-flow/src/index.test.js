// @flow
import handler from '.'

const setup = () => {
  return {
    eventPath: '',
    filename: 'dummy.js',
  }
}

test('return flow definition', () => {
  const css = `
    .a {color: red;}
  `
  const result = handler(css, setup())
  expect(result.code).toMatchSnapshot()
})
