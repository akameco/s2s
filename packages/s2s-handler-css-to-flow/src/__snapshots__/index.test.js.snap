// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`return flow definition 1`] = `
"
// @flow
declare export default {|
  +'a': string,
|}
  "
`;
